pyvista/core/pointset.py

```diff
@@ -1746,6 +1766,12 @@ def _from_arrays(self, x, y, z):
         z : np.ndarray
             Position of the points in z direction.
 
+        force_float : bool, optional
+            Casts the datatype to float32 if points datatype are
+            non-float.  Default ``True``. Set this to ``False`` to
+            allow non-float types, though this may lead to errors when
+            transforming datasets.
```

Comment:
Same as above.

Reply:
```suggestion
            Casts the datatype to ``float32`` if points datatype are
            non-float.  Default ``True``. Set this to ``False`` to allow
            non-float types, though this may lead to intermediate float
            truncation when rotating datasets.
```